### DISCUSSION TITLE
Wiki - Copyright notice

### DISCUSSION THREAD

**Discussion by AWqxKAWERbXo** [leokhoa/laragon#925] [General]
https://github.com/leokhoa/laragon/wiki/_Footer/_compare/9de69b4709d2b17dac0ed6751aaac836a1bddf09...aa1ac1a76537d1a16adc8afa7c0e948adf7da401

@indra87g you added the copyright notice for the laragon wiki, what was the intention?

Imho, there shouldn't be a restricted copyright for these articles. I'd like to know the opinion of @Pen-y-Fan and @Sophist-UK who contributed a lot and restored / rewrited public information. Also @Sunchock and @Vann-Dev @kstingel @michaelhaberle @macarthuror did contribute.

I'd be totally fine if Wiki pages are published under [CC BY 4.0 - Creative Commons Attribution 4.0 International](https://creativecommons.org/licenses/by/4.0/?ref=chooser-v1).

**Comment by Sophist-UK** [comment]
I haven't contributed that much to the Laragon wiki, but what I did contribute is my copyright and the new footer does this.

So I don't have any objection to the copyright part of the notice which states that my contributions are my own copyright, but the part which says "All rights reserved" is not correct - because that means that the right to read and to update this wiki is reserved which cannot be what was intended, and indeed I do NOT reserve some rights to my copyright because I want other people to read it and correct any mistakes and update it as an when it needs updating and in general to benefit from it.

@indra87g is a youngster and I am sure that their heart is in the right place by adding this - and indeed I think it is a good idea because you do want to assert copyright and you do want to disallow certain uses i.e. someone taking it and using it to launch their own copy of Laragon. 

But the license conditions for using copyright materials need to be stated on a wiki page and they need to be reasonable for normal usage, but restrictive against e.g. commercial use.

**Comment by Vann-Dev** [comment]
I'm also not contributed that much, i only fix some URL that pointed into old repository, and my changes was for helping other, and after read all the conversation I agree with them.

Also yes, @indra87g was a youngster, u can see on his bio saying he was 15 y,o so... i think he was thinking to secure it before someone took and launch their own copy of laragon.

Also thanks for tagging me into this conversation 😃

**Comment by Pen-y-Fan** [comment]
I would agree with the creative commons licence. It is the licence I use on my blog too.

**Comment by AWqxKAWERbXo** [comment]
Thank you already for your quick response, maybe @leokhoa could confirm that he would be okay as it's his repository.

**Comment by Sunchock** [comment]
As was said before, I contributed an article to update it with links and a newer version.

In my opinion, it is in this spirit that a wiki must be and must remain, free for users to contribute to it. It will only be richer for everyone
